### PR TITLE
Add index.bs containing only metadata, and generated index.html.

### PR DESCRIPTION
**Pull request by tkent-google**
***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Dec 5, 2019, 11:38 PM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Frawcdn.githack.com%2FWICG%2Fcustom-state-pseudo-class%2F5d8e8d6bd61048e0dda47092a518117148530e90%2Findex.html%3Fforce%3D1%26isPreview%3Dtrue)

```
[33m🕵️‍♀️  That doesn't seem to be a ReSpec document. Please check manually:[39m [36m[Function: url][39m
```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20WICG/custom-state-pseudo-class%231.)._
</details>

### CODE REVIEW

**Comment by tkent-google** [comment]
@domenic @rakina Would you review this please?

File: index.bs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<pre class=metadata>
+Title: Custom State Pseudo Class
+Shortname: custom-state-pseudo-class
+Level: 1
+Status: DREAM
```
Comment (domenic) [inline index.bs]: I think CG-DRAFT is most appropriate in these cases.

File: .pr-preview.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "src_file": "index.html",
-  "type": "respec",
+  "src_file": "index.bs",
+  "type": "bikeshed",
   "params": {
     "force": 1
```
Comment (domenic) [inline .pr-preview.json]: I would remove this; force-ignoring build errors is generally bad.

File: index.bs [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+Group: WICG
+ED: https://wicg.github.io/custom-state-pseudo-class/
+Editor: Kent Tamura, Google https://www.google.com/, tkent@google.com
+Editor: Rakina Zata Amni, Google https://www.google.com/, rakina@google.com
+Editor: Domenic Denicola, Google https://www.google.com/, d@domenic.me, https://domenic.me/
+Repository: https://github.com/WICG/custom-state-pseudo-class/
+Abstract: TODO
+Markup Shorthands: markdown yes
+</pre>
```
Comment (domenic) [inline index.bs:16]: I recommend also `Default Biblio Status: current` and `Complain About: accidental-2119 yes, missing-example-ids yes`

**Review by domenic — APPROVED** [review]
A few minor suggestions.

I can help set up auto-deploy so that you don't have to check in the generated index.html, if I get admin rights for the repo. Probably @cwilso or @yoavweiss will need to grant those, but maybe you can too?

**Comment by travisleithead** [comment]
@domenic Admin rights granted.

**Comment by domenic** [comment]
Awesome! I will set that up after this gets committed.

File: .pr-preview.json (hunk 2 above)
Comment (tkent-google) [reply .pr-preview.json]: Done.

File: index.bs (hunk 1 above)
Comment (tkent-google) [reply index.bs]: Done.

File: index.bs (hunk 3 above)
Comment (tkent-google) [reply index.bs:16]: Done.

**Pull request merged**